projects/igniteui-angular/src/lib/calendar/calendar.component.ts

```diff
@@ -534,6 +547,30 @@ export class IgxCalendarComponent implements OnInit, ControlValueAccessor {
      *@hidden
      */
     private _monthAction = '';
+    /**
+    *@hidden
+    */
+    private _locale = 'en';
```

Comment:
Do not use "_" as a prefix for private properties ([coding-guidelines](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names)). As I see there are other private properties that do not cover this requirement, make changes to them also.

Reply:
Remove prefix for properties:
rangeStarted,  formatterMonth, formatterDay, formatterYear, formatterMonthday and formatterWeekday